downstream/modules/platform/proc-enable-pac.adoc

```diff
@@ -0,0 +1,60 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-08
+:_mod-docs-content-type: PROCEDURE
+
+[id="enable-pac_{context}"]
+= Enabling policy enforcement
+
+During installation, you must configure your {PlatformNameShort} instance to include the policy enforcement feature. You can do this by modifying the feature flag variables in your configuration file.
+Follow the instructions below relevant to your installation type.
+
+== {OCPShort} Installation
+
+For {OCPShort} installations, you must modify the {PlatformNameShort} custom resource. Add the following to the spec section: 
+
+[source,yaml]
+----
+spec: 
+  feature_flags: 
+    FEATURE_POLICY_AS_CODE_ENABLED: True 
+----
+
+After applying the changes, wait for the operator to complete the update process. The operator will automatically handle the necessary service restarts and configuration updates.
```

Comment:
Change second sentence to active voice:

The operator automatically handles the necessary ...